fiftyone/operators/registry.py

```diff
@@ -32,25 +32,25 @@ def get_operator(operator_uri, enabled=True):
     return operator
 
 
-def list_operators(enabled=True, type=None, builtins_only=False):
+def list_operators(enabled=True, builtin="all", type=None):
     """Returns all available operators.
 
     Args:
         enabled (True): whether to include only enabled operators (True) or
             only disabled operators (False) or all operators ("all")
+        builtin (None): whether to include only builtin operators (True) or
+            only non-builtin operators (False) or all operators ("all")
```

Comment:
Purely from a docs pov this is confusing since it lists both None as the default and “all”. These appear functionally equivalent when reviewing the implementation. Changing the default to “all” seems right.

Reply:
yes intended to write `builtin ("all")` in the docs. Will fix now 👍